src/app/header/header.component.scss

```diff
@@ -1,27 +1,88 @@
-.header-cover {
-  background: #3849F9;
-  padding: 1.5rem 5rem;
-  color: #FFFFFF;
-  box-shadow: 0px 7px 19px rgba(0, 0, 0, 0.11);
-  .logo-emphasized {
-    background-color: #FFBD00;
-    color: #3849F9;
-  }
-}
-
-.authorization {
+.authorization{
    display: flex;
    margin:1%;
 }
 
-h1 {
+h1{
    margin:0 auto;
 }
 
-.mat-button {
+.mat-button{
    border:1px solid gray;
 }
 
-.material-icon {
+.material-icon{
    line-height:none;
 }
+.header{
+   padding: 24px 70px 40px 70px;
```

Comment:
did you check it in low resolution?

Reply:
changed to %, also added flex-wrap for low-resolution